issue found on: types/errors.go:111:1: paramTypeCombine

an issue has been detected in source code: ./types/errors.go:111:1: paramTypeCombine: func(regexStr string, str string) string could be replaced with func(regexStr, str string) string

--- a/types/errors.go
+++ b/types/errors.go
@@ -108,7 +108,7 @@ func ConvertDBError(err error, itemID interface{}) error {
 }
 */
 
-func regexGetFirstMatchOrLogError(regexStr string, str string) string {
+func regexGetFirstMatchOrLogError(regexStr, str string) string {
 	return regexGetNthMatchOrLogError(regexStr, 1, str)
 }
 

